vendor/github.com/urfave/cli/help.go

```diff
@@ -82,7 +82,7 @@ OPTIONS:
 var helpCommand = Command{
 	Name:      "help",
 	Aliases:   []string{"h"},
-	Usage:     "Shows a list of commands or help for one command",
+	Usage:     "Show a list of commands or help for one command",
```

Comment:
You need to make this change in the urfave upstream project first and then get that update vendored into this package.  For now just remove this change.